copper-server-api/src/main/java/com/pogeyan/cmis/api/data/ITracingFacade.java

```diff
@@ -0,0 +1,18 @@
+package com.pogeyan.cmis.api.data;
+
+import java.util.Map;
+
+public interface ITracingFacade {
```

Comment:
We just need "startSpan" and "endSpan" APIs, please explain why we need other APIs.